system_tests/bigtable.py

```diff
@@ -356,6 +357,46 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         cell4 = Cell(CELL_VAL4, timestamp4)
         return cell1, cell2, cell3, cell4
 
+    def test_read_large_cell_limit(self):
```

Comment:
This system test is the only one related to the code changes. Before the fixes, this request would fail:

```python
>>> try:
...     partial_row_data = table.read_row(ROW_KEY)
... except GrpcRendezvous as exc:
...     partial_row_data = exc
...
>>> partial_row_data.code()
<StatusCode.INTERNAL: (13, 'internal')>
>>> partial_row_data.details()
'Max message size exceeded'
```

We should probably punt on the other two system tests added and discuss the right path forward.